include/oneapi/dpl/pstl/experimental/internal/for_loop.h

```diff
@@ -56,6 +56,9 @@ template <typename _ExecutionPolicy, typename _Ip, typename... _Rest>
 void
 for_loop(_ExecutionPolicy&& __exec, type_identity_t<_Ip> __start, _Ip __finish, _Rest&&... __rest)
 {
+    static_assert(oneapi::dpl::__internal::__is_host_execution_policy<::std::decay_t<_ExecutionPolicy>>::value,
+                  "for_loop is implemented for the host policies only");
```

Comment:
I'm not sure "host policy" is a well defined term by our documentation (its not present in our guide).  In the past I've been cautioned away from using this terminology in error messages.  
We do however use "device policies", "device execution policies" and "C++ Standard Execution Policies" in the guide.

Reply:
"* The algorithms that destroy data: ``destroy`` and ``destroy_n`` should be called with a host policy when using host data that was initialized on the host, and should be called with a device policy when using device data that was initialized on the device. Otherwise, the result is undefined."

Reply:
Aha, my mistake. those are relatively newly added, and not yet in the version I was searching through.  (and were not there when I was previously cautioned against such terminology).  

To me, while there is this reference in the "known limitations" there is still not a definition of the term, though one could have some reasonable expectation of what it means.  

Maybe the best solution is to add a definition of this term to `execution_policies.rst`.  It is a reasonable shortening or colloquialism of "C++ Standard Execution Policies" for the purposes of oneDPL, I think.